### DISCUSSION TITLE
배송 상태 구독이란?

### DISCUSSION THREAD

**Discussion by wenodev** [CodeSoom/DDD-Kurly-Clone-Order#13] [Q&A] [answered]
- 배송 상태 구독이라는게 어떤 행위인가요?
![image](https://user-images.githubusercontent.com/35131529/138851704-85eaf676-3fe2-46d3-a7a1-ba1ee887eacb.png)

**Comment by joowankim** [comment, marked as answer]
주문에 대한 배송의 상태가 변경될 때 주문의 상태도 함께 변경해야하기 때문에 그 정보를 알기위해 배송에서 발행하는 배송상태의 변경에 대한 이벤트를 구독하는 것을 의미합니다. 그렇게 구현을 하는게 이상적일 것 같아서 일단은 써보았습니다.
